board/safety.h

```diff
@@ -485,13 +485,9 @@ float interpolate(struct lookup_t xy, float x) {
 
 // Safety checks for longitudinal actuation
 bool longitudinal_accel_checks(int desired_accel, const LongitudinalLimits limits) {
-  bool violation = false;
```

Comment:
@sshane can you merge this and make the rest of the tests fail?

Reply:
would definitely fail with old safety and updated tests now

```python
FAIL: test_accel_safety_check (__main__.TestVolkswagenMqbLongSafety)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_volkswagen_mqb.py", line 218, in test_accel_safety_check
    self.assertEqual(send, self._tx(self._acc_06_msg(accel)), (controls_allowed, accel))
AssertionError: True != 0 : (True, 3.01)
```

```
FAIL: test_accel_actuation_limits (__main__.TestVolkswagenPqLongSafety)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/batman/openpilot/panda/tests/safety/common.py", line 169, in test_accel_actuation_limits
    self.assertEqual(should_tx, self._tx(self._accel_msg(accel)))
AssertionError: True != 0
```